src/pages/components/images.mdx

```diff
@@ -15,27 +15,23 @@ import ResourceLinks from 'components/ResourceLinks';
 
 <AnchorLinks>
   <AnchorLink>Overview</AnchorLink>
-  <AnchorLink>Image</AnchorLink>
-  <AnchorLink>Image with caption</AnchorLink>
+  <AnchorLink>Modifiers</AnchorLink>
+  <AnchorLink>Resources</AnchorLink>
+  <AnchorLink>Content guidance</AnchorLink>
   <AnchorLink>Feedback</AnchorLink>
 </AnchorLinks>
 
 ## Overview
 
-There are two image components available. The Image component allows for a simple image on a page. The Image with
-caption component offers the addition of a caption and zoom, which is helpful when showing detailed product shots.
-
-## Image
-
 <Row>
 <Column  colLg={8} colMd={8} colSm={4}>
 
-Image, at its core, is a wrapper that adds an image to the page. You have the option to pass multiple images to the
+Image, at its core, is a wrapper that adds an image to the page. It can display visual assets ranging from photography and renders to illustrations and diagrams.You have the option to pass multiple images to the
```

Comment:
```suggestion
Image, at its core, is a wrapper that adds an image to the page. It can display visual assets ranging from photography and renders to illustrations and diagrams. You have the option to pass multiple images into the
```